test/reportGeneratorTest.py

```diff
@@ -106,22 +112,28 @@ def testCreateManagementReportBody(self):
             the total across the account is displayed
         """
         acct_expenditures = {
-                        '1234': {
-                                    'user1': {'service1': {'2019-01-01': 1.0005, '2019-01-02': 0.0005, 'Total': 1.001},
-                                              'Total': 1.001},
-                                    'user2': {'service1': {'2019-01-01': 0.0005, '2019-01-02': 0.0005, 'Total': 0.001},
-                                              'service2': {'2019-01-01': 0.0005, '2019-01-02': 0.0005, 'Total': 0.001},
-                                              'Total': 0.002},
-                                    'Total': 1.003}
+                        '1234': {'Owner': {
+                                    'user1': {'service1': {'2019-01-01': 1.0005, '2019-01-02': 0.1005, 'Total': 1.101, 'Increase': 0.1005},
+                                              'Total': 1.101, 'Increase': 0.1005},
+                                    'user2': {'service1': {'2019-01-01': 0.0005, '2019-01-02': 0.0005, 'Total': 0.001, 'Increase': 0.0005},
+                                              'service2': {'2019-01-01': 0.0005, '2019-01-02': 0.0005, 'Total': 0.001, 'Increase': 0.0005},
+                                              'Total': 0.002, 'Increase': 0.001},
+                                    'Total': 1.103, 'Increase': 0.1015}}
```

Comment:
I think there needs to not be the 'service1', 'service2' level in this dictionary?